Update terraform readme, typo, and DOKS K8s version

tiffanyfay opened the pull request:
Fixed typo in variables.tf and updated the DOKS slug version 

Updated DOKS kubernetes version to 1.29 in bootstraper.tfvars because when it was 1.25 it threw an error that talked about the latest kubernetes version, but then said the version variable wasn't set. Technically it was set, it just was a version that was no longer supported by DOKS. This happened because the current readme said to update the version in variables.tf instead. Since variables.tf is the definition and has the defaults, it makes more sense for the user to update the boostrapper.tfvars file like the rest of the things they need to change.

Updated the terraform README based on these things etc.
One thing also is the bootstrapped,tfvars has a max nodes that's greater than the default 3 droplet limit. 

I'm open to changing anything you don't agree with @diabhey :).

diabhey commented on infrastructure/terraform/README.md:
````diff
@@ -60,7 +42,45 @@ variable "doks_k8s_version" {
+    ```console
+    doctl k8s options versions
+    ```
+    ```bash
````
@tiffanyfay Merge this block with the ```console``` block on top as this is just the output. Else, it appears as a multiline command that one can copy.

diabhey commented on infrastructure/terraform/README.md:
````diff
@@ -78,7 +98,7 @@ variable "doks_k8s_version" {
 
     ```bash
-    # <cluster-id> can be found in the output of the terraform module
+    # <cluster-id> can be found in the output of the Terraform module
````
it is lowercase `terraform`

diabhey reviewed (commented):
Minor changes. Otherwise LGTM.

tiffanyfay commented on infrastructure/terraform/README.md:
````diff
@@ -60,7 +42,45 @@ variable "doks_k8s_version" {
+    ```console
+    doctl k8s options versions
+    ```
+    ```bash
````
To verify, do you mean to have it be like this?

    ```console
    doctl k8s options versions

    Slug           Kubernetes Version    Supported Features
    1.29.1-do.0     1.29.1                cluster-autoscaler, docr-integration, ha-control-plane, token-authentication
    1.28.6-do.0     1.28.6                cluster-autoscaler, docr-integration, ha-control-plane, token-authentication
    ...
    ```
    
If yes, I changed/split this intentionally because the copy icon copies the text as well and you'll get errors running the whole thing or have to manually highlight just the command. I think the two options that make the most sense here are to either remove the output or to write above it that this is an example output. I don't know that having the output is necessary at all here.

diabhey replied:
@tiffanyfay : yeah remove the console output and just keep the command. Thanks

tiffanyfay replied:
@diabhey Done

tiffanyfay commented:
Looks like it works with the latest ArgoCD Helm chart:
![image](https://github.com/hivenetes/k8s-bootstrapper/assets/12282848/3ac78f00-3a1c-4c1a-942b-7e72d8bc1036)


Also, I updated the readme to not use this GH path for the `bootstrap.yaml` since any changes the user makes to `values.yaml` won't matter and they may not realize it in a rush. And I called out that they need to update the `repoURL` if they make changes. I switched the path vs suggesting they change it so that the copy button works still.
